packages/features/bookings/lib/getBookingFields.ts

```diff
@@ -151,10 +151,12 @@ export const ensureBookingInputsHaveSystemFields = ({
   // These fields should be added before other user fields
   const systemBeforeFields: typeof bookingFields = [
     {
-      defaultLabel: "your_name",
       type: "name",
```

Comment:
Because it has type 'name', `fieldTypesConfigMap` would provide the corresponding config for this type.